src/ConvertInputFormat/main.cpp

```diff
@@ -70,14 +70,12 @@ enum class OutputTypes
     BSON
 };
 
-template <typename T>
-void displayMessage(T t)
+template <typename T> void displayMessage(T t)
```

Comment:
Small non-functional code changes to bring the code up to our Clang-Format style.